addons/website/views/snippets/s_product_catalog.xml

```diff
@@ -47,11 +60,15 @@
 </template>
 
 <template id="s_product_catalog_dish">
-    <li class="s_product_catalog_dish" data-name="Product">
-        <p class="s_product_catalog_dish_title d-flex align-items-baseline pb-2">
-            <span t-esc="name" class="s_product_catalog_dish_name s_product_catalog_dish_dot_leaders"/>
-            <span t-esc="price" class="s_product_catalog_dish_price ms-auto ps-2"/>
+    <li t-attf-class="s_product_catalog_dish {{ not is_last and 'mb-4' }}" data-name="Product">
```

Comment:
`{{ not is_last and 'mb-4' }}` -> We cannot do that because entries can be moved around and new ones created. ;)
What about adding `mb-4` by default ? 
![image](https://github.com/user-attachments/assets/ab033c28-d587-4eb2-921f-6ec0f3524310)